security/security-core/src/main/java/io/camunda/security/configuration/OidcAuthenticationConfiguration.java

```diff
@@ -16,6 +18,7 @@ public class OidcAuthenticationConfiguration {
   private String scope = "openid,profile";
   private String jwkSetUri;
   private String authorizationUri;
+  private Set<String> audience;
```

Comment:
🔧 As this is a collection I prefer the plural
```suggestion
  private Set<String> audiences;
```